include/eve/algo/concepts/iterator_cardinal.hpp

```diff
@@ -55,17 +69,33 @@ namespace eve::algo
   template <typename I>
   constexpr std::ptrdiff_t iterator_cardinal_v = iterator_cardinal_t<I>{}();
 
-
   //================================================================================================
-  //! @addtogroup eve.algo.concepts
+  //! @addtogroup algo_concepts
   //! @{
-  //!  @struct wide_value_type
-  //!  @brief for an instance of `eve::algo::iterator` a shortcut: wide<value_type_t<I>, iterator_cardinal_t<I>>
+  //!   @struct wide_value_type
+  //!   @brief Computes an eve::simd_value from an iterator
   //!
   //!   **Required header:** `#include <eve/algo/concepts.hpp>`
+  //!
+  //!   For a given iterator type `I`, computes the eve::simd_value type associate to the value
+  //!   type referenced byt the iterator `I`.
+  //!
+  //!   @tparam I A type modeling  eve::iterator
```

Comment:
`eve::algo::iterator`
We didn't move the iterator to core.

Reply:
But isn't the concept in eve however ?

Reply:
No: https://github.com/jfalcou/eve/blob/fc060956c10e40463d4ba1111bd573336ec3d5e5/include/eve/algo/concepts/eve_iterator.hpp#L42

Reply:
OK